draft-ietf-rtcweb-jsep.xml

```diff
@@ -3646,11 +3650,28 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
       <t>
         <list>
 
-	  <t>If the RTCP packet has a SDES with a MID, and that MID is
-	  in the table mapping MID to RtpReceiver, update the SSRC
-	  mapping table to include an entry mapping the packet's SSRC
-	  to the RtpReceiver.</t>
-
+	  <t>For each SDES item in the RTCP packet, if the RTCP packet
+	  has a SDES with a MID, and that MID is in the table mapping
+	  MID to a RtpReceiver, update the incoming SSRC mapping table
+	  to include an entry mapping the SSRC for that SDES item to
+	  the RtpReceiver associated with the MID.</t>
+
+	  <t> For each Receiver Report in the RTCP packet, if the SSRC
+	  for the Receiver Report is found in the outgoing SSRC table,
+	  forward a copy of the RTCP packet to the RtpReceiver found
```

Comment:
nit: Elsewhere it looks like we say "deliver", not "forward"

Also, deliver to the RtpSender here, not receiver?